server/README.md

```diff
@@ -1 +0,0 @@
-# Server for Pokedex App
```

Comment:
Please don't delete it.
We should add to there all our API's and more explanation for better documentation.